package com.nataliaF.service.exception;

public class RelatorioNaoEncontradoException extends RuntimeException {
	public RelatorioNaoEncontradoException(Integer id) {
		super("Consulta n√£o encontrado. ID = " + id);
	}
}